mappings/net/minecraft/client/render/model/ModelLoader.mapping

```diff
@@ -1,14 +1,20 @@
 CLASS dvq net/minecraft/client/render/model/ModelLoader
 	CLASS dvq$a ModelLoaderException
-	FIELD A BLOCK_STATE_FACTORY_OVERRIDES Ljava/util/Map;
+	FIELD A STATIC_DEFINITIONS Ljava/util/Map;
 	FIELD B resourceManager Lxd;
 	FIELD C spriteAtlas Ldty;
 	FIELD D modelsToLoad Ljava/util/Set;
 	FIELD E variantMapDeserializationContext Ldle$a;
 	FIELD F unbakedModels Ljava/util/Map;
-	FIELD G modelRotationCache Ljava/util/Map;
+	FIELD G bakedModelCache Ljava/util/Map;
```

Comment:
I think, if it's still rotation-keyed, it's important to keep that in mind.

Reply:
It's keyed by `ModelBakeSettings`, which used to be `ModelRotationContainer`, but actually contains more than just rotation (for example `uvLock`)

Reply:
OK